src/components/utils/isDesktop.js

```diff
@@ -1,3 +1,3 @@
-const width = (window.matchMedia("(min-width: 800px)").matches)
```

Comment:
Might as well get rid of this file entirely, right? If we're gonna use `DetectRTC`, we might as well use it